packages/metro/src/lib/transformHelpers.js

```diff
@@ -71,6 +71,7 @@ async function calcTransformerOptions(
       onProgress: null,
       experimentalImportBundleSupport:
         config.transformer.experimentalImportBundleSupport,
+      optionalDependency: config.resolver.optionalDependency,
```

Comment:
Can we call this `allowOptionalDependencies`?

Reply:
sure